feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -270,15 +270,28 @@ func (tc *testCase) verifyInstall(ctx context.Context, t *testing.T) {
 	}
 
 	if tc.dualSup {
-		if got, want := r.GetVerifyStandby().GetVerifyResponse().GetActivationFailMessage(), ""; got != want {
-			t.Errorf("OS.Verify Standby ActivationFailMessage: got %q, want %q", got, want)
-		}
+		var counter int
+		const maxRetries = 15
+		for i := 1; i < maxRetries; i++ {
+			r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
+			if err != nil {
+				t.Fatalf("OS.Verify request failed: %s", err)
+			}
+			if counter == maxRetries {
+				t.Fatalf("OS.Verify Standby could not obtain version status in 15 minutes")
+			}
+			counter++
```

Comment:
`counters` basically is `i`, right? seems like we don't need this extra `counter` variable?

Reply:
Sure, I'll remove `counter` and keep track of `i`, instead.
I'll update that in next batch commit.